### PR TITLE
Lua Recipes

### PR DESCRIPTION
**Pull request by jerowe**
Lua-luafilesystem
lua-luaposix
lmod

### CODE REVIEW

**Comment by jerowe** [comment]
@rolando, could you take a look at these? I'm not sure why the appveyor is building. I am trying to skip that, at least for lmod.

**Comment by rolando** [comment]
@jerowe hey, I'm just a casual contributor :)

I'm not familiar with lua building process so can't really tell if something is missing. You can ping @ocefpaf, @jakirkham or any other @conda-forge/core for direct feedback.

**Comment by jerowe** [comment]
Alright. Thanks again! I think I was missing a skip # [win], so we'll see.

**Comment by jerowe** [comment]
ping @boegel

File: recipes/bcolz/meta.yaml [hunk 1]
```diff
@@ -1,56 +0,0 @@
-{% set name = "bcolz" %}
```
Comment (scopatz) [inline recipes/bcolz/meta.yaml]: Same here, please restore other recipes to the state they are on master. Thanks!

**Comment by jerowe** [comment]
@jakirkham , I rebased against master branch.

**Comment by jerowe** [comment]
@jakirkham , merge?

**Comment by scopatz** [comment]
LGTM thanks

**Pull request merged**